consensus/state.go

```diff
@@ -1641,21 +1650,24 @@ func (cs *ConsensusState) addVote(vote *types.Vote, peerID p2p.ID) (added bool,
 	case types.VoteTypePrecommit:
 		precommits := cs.Votes.Precommits(vote.Round)
 		cs.Logger.Info("Added to precommit", "vote", vote, "precommits", precommits.StringShort())
+
 		blockID, ok := precommits.TwoThirdsMajority()
-		if ok && len(blockID.Hash) != 0 {
-			// Executed as TwoThirdsMajority could be from a higher round
```

Comment:
Isn't this comment still true?

Reply:
We could maybe move comment before enterNewRound, L1656.